net-mgmt/pfSense-pkg-softflowd/files/usr/local/pkg/softflowd.inc

```diff
@@ -0,0 +1,163 @@
+<?php
+/*
+ * softflowd.inc
+ *
+ * part of pfSense (https://www.pfsense.org)
+ * Copyright (c) 2013-2017 Rubicon Communications, LLC (Netgate)
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+require_once("config.inc");
+require_once("globals.inc");
+require_once("interfaces.inc");
+require_once("service-utils.inc");
+require_once("util.inc");
+
+function sync_package_softflowd() {
+	global $config;
+
+	if (is_array($config['installedpackages']['softflowd']['config'][0])) {
+		$cf = $config['installedpackages']['softflowd']['config'][0];
+	} else {
+		$cf = array();
+	}
+	$interface_list = explode(",", $cf['interface']);
+	if (!empty($cf['host']) && !empty($interface_list) && $cf['enable'] != "off") {
+		conf_mount_rw();
```

Comment:
Instead of call conf_mount_rw() inside if and else statements, just move it to a single call outside